.travis.yml

```diff
@@ -91,8 +91,13 @@ install:
 script:
     - build_wheel apache-beam-${VERSION} $PLAT
 
-after_success:
-    - ls -lh ${TRAVIS_BUILD_DIR}/wheelhouse/
-
-    - source deploy_travis.sh
-    - deploy
+deploy:
+  provider: gcs
+  access_key_id: ${ACCESS_KEY_ID}
+  secret_access_key: ${SECRET_ACCESS_KEY}
+  bucket: "beam-wheels-staging"
```

Comment:
Is it possible to stage them under a directory with the version in the name?

Reply:
I looked into this when I wrote this change, there wasn't an obvious way to do this. Directory structures are copied as is.